apps/search/src/components/MainInstructions.tsx

```diff
@@ -14,33 +14,6 @@ export const MainInstructions = ({ children, ...props }: PropsWithBox<{}>) => {
         Once the App is configured, you will be able to perform initial index of your existing
         Saleor database.
       </Text>
-      <Text as="p">
```

Comment:
Removed this section, because just below it theres a list of all of the app webhooks.